src/command.zig

```diff
@@ -35,119 +39,227 @@ const CommandOption = struct {
-        .Use => try use_version(command.param),
-        .Default => try set_default(),
+        .Use => try use_version(command.subcmd, command.param),
+        .Remove => try remove_version(command.subcmd, command.param),
+        // .Default => try set_default(),
         .Version => try get_version(),
         .Help => try display_help(),
         .Unknown => try handle_unknown(),
     }
 }
 
-fn handle_alias(params: []const []const u8) !void {
+/// handle alias, now only support zig
+pub fn handle_alias(params: []const []const u8) !void {
+    if (builtin.os.tag == .windows) return;
+
+    var is_zls: bool = undefined;
+
+    const basename = std.fs.path.basename(params[0]);
+    if (tools.eql_str(basename, "zig")) {
+        is_zls = false;
+    } else if (tools.eql_str(basename, "zls")) {
+        is_zls = true;
+    } else return;
+
     var arena = std.heap.ArenaAllocator.init(tools.get_allocator());
     defer arena.deinit();
 
     const allocator = arena.allocator();
 
     const new_params = try allocator.dupe([]const u8, params);
 
-    const home = tools.get_home();
-    const current_zig_path = try std.fs.path.join(allocator, &.{ home, ".zm", "current", "zig" });
+    const current = try if (is_zls) tools.get_zvm_current_zls(allocator) else tools.get_zvm_current_zig(allocator);
+    const current_path = try std.fs.path.join(allocator, &.{ current, if (is_zls) "zls" else "zig" });
 
-    std.fs.accessAbsolute(current_zig_path, .{}) catch |err| {
+    std.fs.accessAbsolute(current_path, .{}) catch |err| {
         if (err == std.fs.Dir.AccessError.FileNotFound) {
-            std.debug.print("Zig has not been installed yet, please install zig with zvm!\n", .{});
+            std.debug.print("{s} has not been installed yet, please install it fist!\n", .{if (is_zls) "Zls" else "Zig"});
             std.process.exit(1);
         }
         return err;
     };
 
-    new_params[0] = current_zig_path;
+    new_params[0] = current_path;
     return std.process.execv(allocator, new_params);
 }
 
-fn handle_list() !void {
+fn handle_list(param: ?[]const u8) !void {
     const allocator = tools.get_allocator();
-    var version_list = try versions.VersionList.init(allocator);
-    defer version_list.deinit();
 
-    for (version_list.slice()) |version| {
+    const version_list: [][]const u8 = blk: {
+        if (param) |p| {
+            // when zls
+            if (tools.eql_str(p, "zls")) {
+                const res = try tools.http_get(allocator, config.zls_url);
+                defer allocator.free(res);
+
+                var zls_meta = try meta.Zls.init(res, allocator);
+                defer zls_meta.deinit();
+
+                const version_list = try zls_meta.get_version_list(allocator);
+                break :blk version_list;
+            } else
+            // when not zig
+            if (!tools.eql_str(p, "zig")) {
+                std.debug.print("Error param, you can specify zig or zls\n", .{});
```

Comment:
On a general note, shouldn't we use `std.log` instead? We can tackle this in another PR.